pkg/internal/server/grpc/server.go

```diff
@@ -40,10 +41,18 @@ import (
 const (
 	HandlerTraceLogLevel         = 4
 	HandlerDetailedTraceLogLevel = 5
+
+	HandlerDefaultMaxStreamDuration = time.Minute * 60
```

Comment:
```suggestion
	HandlerDefaultMaxStreamDuration = time.Minute * 10
```
This should be 10 minutes ? 
the same value as `defaultMaxStreamingDurationMin = 10`